tests/unit/test_bigbang.py

```diff
@@ -18,6 +19,7 @@
 
 test_txt = ""
 TEMP_DIR = os.path.join(CONFIG.test_data_path, "tmp")
+project_directory = str(Path(os.path.abspath(__file__)).parent.parent.parent)
```

Comment:
I'm not sure a project directory variable is needed, but if it is, maybe we could describe it as a constant like `TEMP_DIR`.

Reply:
No I can get rid of it and use the `CONFIG` for that as suggested above :relaxed: